analytics/stats.py

```diff
@@ -0,0 +1,40 @@
+import json
+import pandas as pd
+from departements import to_departement_number, import_departements
```

Comment:
Personnellement, le script ne fonctionnait pas. J'ai du ajouter `scraper` devant départements
```suggestion
from scraper.departements import to_departement_number, import_departements
```